bindings/nodejs/examples/client/offline_signing/3_send_block.ts

```diff
@@ -1,4 +1,4 @@
-// Copyright 2021-2022 IOTA Stiftung
+// Copyright 2021-2023 IOTA Stiftung
```

Comment:
We leaving these as ranges, or do we want to match the others?

Reply:
I would also like to know that :trollface:

Reply:
no one has a clue ----> resolved :joy: